discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/DeleteDocumentResponse.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.discovery.v1.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * DeleteDocumentResponse.
+ */
+public class DeleteDocumentResponse extends GenericModel {
+
+  /** The unique identifier of the document. */
+  @SerializedName("document_id")
+  private String documentId;
+  /** Status of the document. A deleted document has the status deleted. */
+  private String status;
```

Comment:
as discussed in slack, I think this should be all or none. I would lean towards specifying the enum especially if it is in the swagger. we should pick and choose which enums to generate (like language) and which ones to not (like status)